common/mariadb/values.yaml

```diff
@@ -17,8 +17,8 @@ query_cache_type: "0"
 join_buffer_size: "4M"
 binlog_format: "MIXED"
 expire_logs_days: 10
-character_set_server: 'utf8'  # Should be utf8mb4, but we started with this
-collation_server: 'utf8_general_ci'
+character_set_server: "utf8" # Should be utf8mb4, but we started with this
+collation_server: "utf8_general_ci"
```

Comment:
i hope this does not trigger a hash change and therefor a mariadb restart :)

Reply:
Argh the wonders of auto-formatting.